packages/@react-spectrum/meter/docs/Meter.mdx

```diff
@@ -0,0 +1,121 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/meter';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {Flex} from '@react-spectrum/layout';
+import {Meter} from '@react-spectrum/meter';
+```
+
+# Meter
+
+<p>{docs.exports.Meter.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Meter']}
+  sourceData={[{type: 'Spectrum', url: 'https://spectrum.adobe.com/page/meter/'}]} />
+
+## Example
+
+```tsx example
+<Meter label="Storage space" variant="positive" value={35} />
+```
+
+## Value
+
+Meters are controlled with the `value` prop representing current percentage (as an integer).
```

Comment:
is that true? it must be an integer? or is that just a convenience with our defaults for min/max?

Reply:
i guess not! will adjust the others as well. technically a number (not an int) and will not default min/max